Mtreadwa/faq updates (#68)
stjude/sjcloud-docs: docs/faq.md

[What clinical information is available about samples in St. Jude Cloud?](#what-clinical-information-is-available-about-samples-in-st-jude-cloud)  
[Can I get a copy of IRB consent forms?](#can-i-get-a-copy-of-irb-consent-forms)  
[Can I request FASTQ files on St. Jude Cloud?](#can-i-request-fastq-files-on-st-jude-cloud)  

**Technical Questions**  
[How can I explore and manipulate data files stored on the cloud without downloading the files to my local machine?](#how-can-i-explore-and-manipulate-data-files-stored-on-the-cloud-without-downloading-the-files-to-my-local-machine)    
[How can I connect to DNAnexus API via SSH on a Windows machine?](#how-can-i-connect-to-dnanexus-api-via-ssh-on-a-windows-machine)  
[Why am having trouble connecting to DNAnexus API via SSH?](#why-am-i-getting-a-connectivity-error-when-connecting-to-dnanexus-api-via-ssh)  

**Publication Questions**  
[How do I cite St. Jude Cloud?](#how-do-i-cite-st-jude-cloud)  
[When can I publish my findings using St. Jude Cloud data?](#when-can-i-publish-my-findings-using-st-jude-cloud-data)  

> [Where can I find the Embargo Date?](#where-can-i-find-the-embargo-date)

Reviewer: ```suggestion
[Where can I find the embargo date?](#where-can-i-find-the-embargo-date)
```

**Publication Questions**  
[How do I cite St. Jude Cloud?](#how-do-i-cite-st-jude-cloud)  
[When can I publish my findings using St. Jude Cloud data?](#when-can-i-publish-my-findings-using-st-jude-cloud-data)  
[Where can I find the Embargo Date?](#where-can-i-find-the-embargo-date)

**Miscellaneous**   
[Will St. Jude Cloud host my institution's data in the data browser or on PeCan?](#will-st-jude-cloud-host-my-institutions-data-in-the-data-browser-or-on-pecan)  

### Where can I find the Terms of Service or the Privacy Policy?

You can find the Terms of Service
[here](https://stjude.cloud/terms-of-use.html) and the Privacy Policy
[here](https://platform.stjude.cloud/privacy).

[msgen]: https://azure.microsoft.com/en-us/services/genomics/
[gvcf-spec]: https://gatkforums.broadinstitute.org/gatk/discussion/11004/gvcf-genomic-variant-call-format
[gvcf-diff-from-vcf]: https://gatkforums.broadinstitute.org/gatk/discussion/4017/what-is-a-gvcf-and-how-is-it-different-from-a-regular-vcf

### How can I sign up for updates when new data or features are added to the cloud?

> We are always adding data to St. Jude Cloud, and if you would like to sign up for updates [subscribe here](https://hospital.stjude.org/apps/forms/fb/st-jude-cloud-subscribe/).

Reviewer: ```suggestion
We are always adding data to St. Jude Cloud, and if you would like to sign up for updates, [subscribe here](https://hospital.stjude.org/apps/forms/fb/st-jude-cloud-subscribe/).
```

[here](https://platform.stjude.cloud/access_form).

### Where do I submit the Data Access Agreement (DAA)?
You can submit your Data Access Agreement in the drag and drop box on the [last step of the data request process](./guides/data/data-request.md#requesting-data).

### What if I did not fill out the Data Download Permission section of the original DAA, but now I want to download data?
Since this would be a change in terms from the original agreement, you would need to fill out a new DAA (including the [Data Download Permission section](./guides/forms/how-to-fill-out-DAA.md#data-download-permission) for any datasets you want to download.

### What clinical information is available about samples in St. Jude Cloud?
Currently the only [clinical information](./guides/data/metadata.md#clinical-and-phenotypic-information) we provide is age at diagnosis, diagnosis, ethnicity, and sex. Unfortunately, even if we do collect other information, such as other supportive oncology data, it is not available on the cloud at this time. We are working towards being able to provide additional clinical annotations in the future.

### Can I get a copy of IRB consent forms?

> Unfortunately, we will not be able to share blank consent forms at the current time. We have chosen to remain consistent with the requirements of the other major genomic data repositories in that (1) there is an internal vetting process by the St. Jude IRB to ensure samples may be shared with the research community, but (2) we do not share the informed consents with data requestors. It is important to remember that St. Jude Cloud is the platform upon which all St. Jude data is shared. This means that there are more than 100 consent forms + revisions for the various studies across St. Jude. Thus, there is an additional logistical barrier in that we simply don't have the bandwidth to pull together a packet containing all of this information for each requestor.

Reviewer 2: This is minor, but change "requestors" to "requesters" to be consistent, since we're already using "requesters" in line 86. Based on [this](https://english.stackexchange.com/questions/29254/whats-the-difference-between-requester-and-requestor), I think "requesters" is correct for the documentation.

### Can I request FASTQ files on St. Jude Cloud?

> We do not share FASTQ formats, but several tools exists that you can leverage to revert BAM to FASTQ files. (We recommend using Picard SamToFastq to revert BAM files.) You can efficiently revert BAMs to FASTQs in the cloud by wrapping the conversion tool of your choice into a [Cloud App](./guides/data/creating-a-cloud-app.md).

Reviewer: ```suggestion
We do not share FASTQ formats, but several tools exist that you can leverage to revert BAM to FASTQ files. (We recommend using Picard SamToFastq to revert BAM files.) You can efficiently revert BAMs to FASTQs in the cloud by wrapping the conversion tool of your choice into a [Cloud App](./guides/data/creating-a-cloud-app.md).
```

### How can I explore and manipulate data files stored on the cloud without downloading the files to my local machine?
You can quickly and easily interact with data files using the DNAnexus cloud workstation app. See [this guide](https://wiki.dnanexus.com/Developer-Tutorials/Cloud-Workstations) to help you setup and run the app. Note that if you are doing any type of large-scale, multi-sample analysis, espescially if you plan to repeat the analysis or want to run in parallel, you will want to [write your own cloud app](./guides/data/creating-a-cloud-app.md) rather than use the cloud workstation app.

### How can I connect to DNAnexus API via SSH on a Windows machine?

> To connect via SSH on a Windows machine we reccomend using Windows Subsystem for Linux (WSL) or a Linux virtual machine.  

Reviewer: ```suggestion
To connect via SSH on a Windows machine we recommend using Windows Subsystem for Linux (WSL) or a Linux virtual machine.  
```

The WSL method:  
1. Open PowerShell as an administrator and run   
[Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10#install-the-windows-subsystem-for-linux)  
2. Restart computer.  
3. Install Ubuntu from the [Microsoft Store](https://www.microsoft.com/store/p/ubuntu/9nblggh4msv6).  
4. Open Ubuntu. This prompts you to create a user and password.  
5. Install dx toolkit:  
  `$ sudo apt update`  
  `$ sudo apt install python-minimal python-pip`    
  `$ pip install dxpy`    
6. Close Ubuntu window. Open Ubuntu again.  

> 7. Log in to dnanexus, where $TOKEN is an [API Token](https://wiki.dnanexus.com/Command-Line-Client/Login-and-Logout#Authentication-Tokens).  

Reviewer: ```suggestion
7. Log in to DNAnexus, where `$TOKEN` is an [API Token](https://wiki.dnanexus.com/Command-Line-Client/Login-and-Logout#Authentication-Tokens).  
```

You may also find [these instructions](https://wiki.dnanexus.com/Developer-Tutorials/Connecting-to-Jobs#Setting-up-SSH-on-Windows) on the DNAnexus wiki helpful.

### Why am I getting a connectivity error when connecting to DNAnexus API via SSH?
If you are trying to run something like    
  `$ dx run --ssh <executable> `   
and are getting a connectivity error, it may be that your firewall is too restrictive. Are you able to perform the command from an unrestricted network (like a home network)? If yes, you can resolve this issue by asking your network administrator to whitelist connections to Azure US West. All subnets (Region Name="uswest") are provided [here](https://www.microsoft.com/en-us/download/details.aspx?id=41653).

### How do I cite St. Jude Cloud?
We are currently in progress of preparing a paper for St. Jude Cloud. In the meantime, please refer to the [citation table](index.md#citing-st-jude-cloud) on the Welcome page.

### When can I publish my findings using St. Jude Cloud data?
Once the [embargo date](glossary.md#embargo-date) for the St. Jude datasets that you've used in your reserach has passed, you are legally permitted to publish results based on that data.

> ### Where can I find the Embargo Date?

Reviewer: ```suggestion
### Where can I find the embargo date?
```

*  You will be charged for any *derivative* files stored on the St. Jude Cloud, such as results files obtained through running analyses workflows.
* There is a small monthly storage fee associated for any of your own data you upload to the cloud.
* If you elect to *download* any data from St. Jude Cloud, you will be
  charged an egress fee by DNAnexus. This fee is usually negligible
  unless you are downloading entire cohorts. We are actively
  investigating ways to minimize or eliminate these costs.

> * If you run any of our analysis workflows (such as Rapid RNA-Seq, WARDEN, etc) or your own workflows that you have uploaded and packaged into the cloud, you will be charged for the

Reviewer: ```suggestion
 * If you run any of our analysis workflows (such as Rapid RNA-Seq, WARDEN, etc.) or your own workflows that you have uploaded and packaged into the cloud, you will be charged for the
```

  compute resources used in producing the results. Soon we hope to be able to sponsor all compute costs associated with running our St. Jude Cloud workflows.

You can find DNAnexus's specific resource-based pricing table by navigating to the Billing Account tab of your profile page on DNAnexus and then clicking the green 'Add Billing Info' button next to your name.

### How can I set up billing for my lab?

> Billing setup is different based on whether you are an interval user (you work at St. Jude) or an external user. External users refer to the [Create an Account page](create-an-account.md) for instructions. Internal users search 'Bioinformatics self-service on St. Jude Cloud' from the [intranet home page](https://home.stjude.org/Pages/default.aspx) for instructions. 

Reviewer: ```suggestion
Billing setup is different based on whether you are an internal user (i.e., you work at St. Jude) or an external user. External users should refer to the [Create an Account page](create-an-account.md) for instructions. Internal users can search 'Bioinformatics self-service on St. Jude Cloud' from the [intranet home page](https://home.stjude.org/) for instructions. 
```

### Why do I need to sign the Data Access Agreement (DAA)?
Although the [DAA](glossary.md#data-access-agreement) serves many purposes, the terms included in the data access
agreement are ultimately in place to protect our patients. We take
patient security very seriously, and we require that requesters are
committed to protecting that privacy to the fullest extent.

### Can I make edits/revisions to the DAA?
As a rule, we do not negotiate the terms of the data access agreement unless terms are found to be in conflict with the institution's state law. The terms included in the data access agreement are ultimately in place to protect our patients, a matter which we take very seriously.

> ### Can I get access to a word version of the DAA?

Reviewer: ```suggestion
### Can I get a Microsoft Word version of the DAA?
```

[here](https://platform.stjude.cloud/access_form).

### Where do I submit the Data Access Agreement (DAA)?
You can submit your Data Access Agreement in the drag and drop box on the [last step of the data request process](./guides/data/data-request.md#requesting-data).

### What if I did not fill out the Data Download Permission section of the original DAA, but now I want to download data?
Since this would be a change in terms from the original agreement, you would need to fill out a new DAA (including the [Data Download Permission section](./guides/forms/how-to-fill-out-DAA.md#data-download-permission) for any datasets you want to download.

### What clinical information is available about samples in St. Jude Cloud?
Currently the only [clinical information](./guides/data/metadata.md#clinical-and-phenotypic-information) we provide is age at diagnosis, diagnosis, ethnicity, and sex. Unfortunately, even if we do collect other information, such as other supportive oncology data, it is not available on the cloud at this time. We are working towards being able to provide additional clinical annotations in the future.

### Can I get a copy of IRB consent forms?

> Unfortunately, we will not be able to share blank consent forms at the current time. We have chosen to remain consistent with the requirements of the other major genomic data repositories in that (1) there is an internal vetting process by the St. Jude IRB to ensure samples may be shared with the research community, but (2) we do not share the informed consents with data requestors. It is important to remember that St. Jude Cloud is the platform upon which all St. Jude data is shared. This means that there are more than 100 consent forms + revisions fro the various studies across St. Jude. Thus, there is an additional logistical barrier in that we simply don't have the bandwidth to pull together a packet containing all of this information for each requestor.

Reviewer: ```suggestion
Unfortunately, we will not be able to share blank consent forms at the current time. We have chosen to remain consistent with the requirements of the other major genomic data repositories in that (1) there is an internal vetting process by the St. Jude IRB to ensure samples may be shared with the research community, but (2) we do not share the informed consents with data requestors. It is important to remember that St. Jude Cloud is the platform upon which all St. Jude data is shared. This means that there are more than 100 consent forms + revisions for the various studies across St. Jude. Thus, there is an additional logistical barrier in that we simply don't have the bandwidth to pull together a packet containing all of this information for each requestor.
```

### Can I request FASTQ files on St. Jude Cloud?

> We do not share FASTQ formats, but several tools exists that you can leverage to revert BAM to FASTQ files. (We recommend using Picard SamToFastq to revert BAM files.) You can efficiently revert BAMS to FASTQ in the cloud by wrapping the conversion tool of your choice into a [Cloud App](./guides/data/creatings-a-cloud-app.md).

Reviewer: ```suggestion
We do not share FASTQ formats, but several tools exists that you can leverage to revert BAM to FASTQ files. (We recommend using Picard SamToFastq to revert BAM files.) You can efficiently revert BAMs to FASTQs in the cloud by wrapping the conversion tool of your choice into a [Cloud App](./guides/data/creatings-a-cloud-app.md).
```

The WSL method:  
1. Open PowerShell as an administrator and run   
[Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10#install-the-windows-subsystem-for-linux)  
2. Restart computer.  
3. Install Ubuntu from the [Microsoft Store](https://www.microsoft.com/store/p/ubuntu/9nblggh4msv6).  
4. Open Ubuntu. This prompts you to create a user and password.  
5. Install dx toolkit:  
  `$ sudo apt update`  
  `$ sudo apt install python-minimal python-pip`    
  `$ pip install dxpy`    
6. Close Ubuntu window. Open Ubuntu again.  
7. Log in to dnanexus, where $TOKEN is an [API Token](https://wiki.dnanexus.com/Command-Line-Client/Login-and-Logout#Authentication-Tokens).  
  `$ dx login --token $TOKEN` 

You may also find [these instructions](https://wiki.dnanexus.com/Developer-Tutorials/Connecting-to-Jobs#Setting-up-SSH-on-Windows) on the DNAnexus wiki helpful.

### Why am I getting a connectivity error when connecting to DNAnexus API via SSH?
If you are trying to run something like    

>   `$ dx run \<executable\> --ssh`   

Reviewer: ```suggestion
  `$ dx run --ssh <executable>`   
```
